src/rollup/rollup.ts

```diff
@@ -51,6 +52,8 @@ export async function rollupInternal(
 	);
 	initialiseTimers(inputOptions);
 
+	/** WASM_INIT_PLACEHOLDER */
```

Comment:
The wasm needs to be instantiated, but it is an async process, so I have to put it in an async function(the browser format of Rollup is umd that does not support top-await), I think `rollupInternal` async function is an appropriate place.

Reply:
I am not super fond of using the replace plugin as it needs to scan every single file for one replace, and in your setup several times. Replacing imports is usually much cheaper in compile time. Here, you could supply a dummy init function for NodeJS that is replaced with the WASM init function in the browser build.

Reply:
done